web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div v-page-title="meta.name">
-    <meta-editor
-      v-if="edit && Object.entries(meta).length"
-      :schema="schema"
-      :model="meta"
-      :create="create"
-      @close="edit = false"
-    />
+  <div
+    v-if="schema"
+    v-page-title="meta.name"
+  >
+    <template v-if="edit && Object.entries(meta).length">
+      <meditor
+        v-if="toggles.DJANGO_API"
```

Comment:
This isn't really documented anywhere sadly, but `DJANGO_API` is registered as a property available to all components: https://github.com/dandi/dandiarchive/blob/master/web/src/featureToggle.ts#L28-L34

I did this so that components could avoid doing exactly this. However I thought it was weird to write `if (this.DJANGO_API) {...}` in the component scripts, so I still import `featureToggle` so that I can use `toggles.DJANGO_API` instead. Sorry for that magic.

Reply:
I'm a bit confused, are you recommending a change here? I actually think I used `toggles.DJANGO_API` on purpose, because I thought that in the future the mixin might go away. If you think there's something fundamentally wrong with this, I can change it.

Reply:
My recommendation is:
```suggestion
        v-if="DJANGO_API"
```
and remove `toggles` from the return value in `setup()`. My intention was for the mixin to be permanent; if we ever added a new toggle, we would also add it to the mixin.

Now that I think about it though, maybe it makes more sense to expose `toggles` in the mixin rather than `DJANGO_API` directly. I was mostly trying to avoid having to include that `toggles` in the return value of `setup()` so that the footprint of using a toggle would be as small as possible.

If you think that's a bad pattern, I'm open to changing it.

Reply:
I think the idea of reducing the footprint of the feature toggles is a good one. Since right now the only toggle we have is `DJANGO_API`, and that'll be going away soon, I can just use the mixin value.

Reply:
I actually forgot about one thing which I'd like to point out. Currently, anywhere we use feature toggles we still import `toggles` in the component script, to make any use of it. I don't think we actually _need_ to do this, since the computed prop should still be present via the mixin, but if we don't import and use it that way eslint will yell at us. I don't quite like that we're using the computed value from the mixin in the templates, and importing it for the script. I'll change this to be consistent with the rest of the app, but just thought I'd make my concerns known.